_studio/shared/umc/codec/vp9_dec/src/umc_vp9_utils.cpp

```diff
@@ -282,10 +282,9 @@ namespace UMC_VP9_DECODER
         else
         {
             const int32_t data = GetSegData(seg, segmentId, UMC_VP9_DECODER::SEG_LVL_ALT_Q);
-            return
-                 seg.absDelta == SEGMENT_ABSDATA ?
-                 data :  // Abs value
-                 clamp(baseQIndex + data, 0, MAXQ);  // Delta value
+            const int seg_qindex =
```

Comment:
Please, align style and use ```int32_t``` type

Reply:
Thanks for your comment, I updated the patch to use `int32_t`